tests/rbd_mirror/test_rbd_mirror_journal_to_snap.py

```diff
@@ -5,6 +5,41 @@
 log = Log(__name__)
 
 
+def test_write_from_secondary(self, mirrormode, imagespec):
+    """Method to verify write operation should not be performed from secondary site,
+    as mirrored image got locked from primary site.
+
+    Args:
+        self: mirror2 object
+        mirrormode: type of mirroring journal or snapshot
+        imagespec: poolname + "/" + imagename
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+    """
+    out, err = self.exec_cmd(
+        cmd=(
+            "rbd bench --io-type write --io-threads 16 "
+            f"--io-total '500M' {imagespec}"
+        ),
+        all=True,
+        check_ec=False,
+    )
+    log.debug(err)
+
+    if "Read-only file system" in err:
+        log.info(
+            "As Expected writing data from secondary got failed, "
+            f"mirrored image got locked from primary in {mirrormode} based mirroring"
+        )
+        return 0
+
+    log.error(
+        f"In {mirrormode} based able to write data from secondary, even mirrored image got locked from primary"
```

Comment:
Could you please change this to, 
**Able to write data into secondary image for {mirrormode} based mirroring. This is invalid since image should be locked at primary**

I felt this seems more easily understandable. Please let me know your thoughts.

Reply:
Done, Thanks